src/components/EntityStatus/EntityStatus.js

```diff
@@ -121,15 +119,13 @@ class EntityStatus extends React.Component {
                     {this.renderLink()}
                 </span>
                 {hasClipboardButton && (
-                    <Button
-                        component="span"
-                        size="s"
+                    <ClipboardButton
```

Comment:
`ClipboardButton` in tables looks different compared to other buttons in extended version

Nodes table (old, with developer ui link)
<img width="356" alt="Screen Shot 2024-01-15 at 19 51 53" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/6ce13cdf-f6b4-4b8c-81cf-c77f1d773c0b">

Databases table (old, with monitoring link)
<img width="240" alt="Screen Shot 2024-01-15 at 19 51 39" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/5950841e-667c-41ca-95cc-adb994bdf254">

Databases table (new version, without additional buttons)
<img width="143" alt="Screen Shot 2024-01-15 at 19 52 37" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/ad76dae1-97f4-4370-b8aa-d25960a52d93">

Reply:
Refactored clipboard buttons a bit to visually unify them with Gravity UI.

Reply:
Should looks similar now because both use Graviti's `Button`.